integration/spark/build.gradle

```diff
@@ -154,6 +156,7 @@ dependencies {
 
 sourceSets {
     main.java.srcDirs = ["src/main/common/java", "src/main/spark2/java"]
+    test.java.srcDirs = ["src/test/common/java"]
```

Comment:
Shouldn't we have `spark2` here, just as in `main.java.srcDirs`? It looks like it works though.

Reply:
I was thinking about it but at the moment we do not have any Spark2 test that fails for Spark3 and all the existing tests are put to `common`. So we could add this here but spark2 submodule will not be pushed to repo as it would be just an empty directory. Also, I was not sure if we are going to have any new Spark2 tests in future.

Reply:
All the current tests are within a common submodule as we do not have any Spark2 specific tests (tests that do not  compile for Spark 3)